Block/Js.php

```diff
@@ -226,6 +226,7 @@ public function getSettings()
             'is_pre_auth'              => $this->getIsPreAuth(),
             'default_error_message'    => $this->getBoltPopupErrorMessage(),
             'button_css_styles'        => $this->getButtonCssStyles(),
+            'always_present_checkout' => $this->featureSwitches->isAlwaysPresentCheckoutEnabled(),
```

Comment:
You've used feature switches here which is good, but something like this should also be controllable by the merchant in their plugin settings. Look at `pay_by_link_url` a few lines above for an example. Can you add that behaviour as well so it's toggle-able in M2?

Reply:
To do this do I just add a config path and call something like `$this->getCustomURLValueOrDefault(self::ALWAYS_PRESENT_CHECKOUT_CONFIG_PATH, false);` and define `ALWAYS_PRESENT_CHECKOUT_CONFIG_PATH = "payments/boltpay/always_present_checkout"`? Is there anywhere to define a toggle-able switch that shows up in setings panel?

Reply:
Yes, in `etc/config.xml`.

Reply:
You also need it in `etc/adminhtml/system.xml`.

Reply:
Fixed up